spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/spring-cloud-gcp-integration-pubsub-sample-receiver/pom.xml

```diff
@@ -6,7 +6,7 @@
 		<!-- Your own application should inherit from spring-boot-starter-parent -->
 		<artifactId>spring-cloud-gcp-integration-pubsub-sample</artifactId>
 		<groupId>com.google.cloud</groupId>
-		<version>3.4.1-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+		<version>4.0.0-SNAPSHOT</version>
```

Comment:
This comment `<!-- {x-version-update:spring-cloud-gcp:current} -->`may be needed to manage the versions via release-please. Reference: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1397/files

Reply:
done